### PR TITLE
refine ci coverage

### PR DESCRIPTION
**Pull request by jianlinliu**
Follow up https://github.com/openshift/release/pull/34057, continue to refine ci coverage.

1. drop {vsphere|azure|gcp}-upi-{disconnected|proxy} ci coverage (got confirm form PMs.)
2. drop duplicated coverage for 4.13 just like what we did in https://github.com/openshift/release/pull/34057
3. drop duplicated `gcp-ipi-disconnected`  and `azure-ipi-private-sdn`ci coverage
4. lower test frequency for p1 ci to avoid daily trigger, we can amend them latter.  
5. adjust `e2e-azure-ipi-resourcegroup` test frequency

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @zhaozhanqi @gpei @aleskandro @jianli-wei @jinyunma @yunjiang29 @shellyyang1989 @anuragthehatter @liangxia @jhou1 @MayXuQQ for review.

**Comment by MayXuQQ** [comment]
have doc about the priority level and the cron value (period) relation?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -2201,8 +1933,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-upi-xpn
-- as: e2e-ibmcloud-ipi-p1
```
[the remark this answers is not in the document]
Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2204]: p1/p2/p3 is the priority, comes from doc "Integration Test Configuration Matrix", we shall not change it.
Lets get this PR and https://github.com/openshift/release/pull/34095 merge quickly, then use follow up PR(s) to update that to ibmcloud-ipi-p1-fX (fX can be f1 (test every day), f2 (test every 2 days), f3(test every 3 days), ..., based on the test requirement from feature team and/or profile owner. HDYT ?

[the remark this answers is not in the document]
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2204]: Yeah, once we introduce `f` tag name (WIP in https://github.com/openshift/release/pull/34095), I will update the again.

[the remark this answers is not in the document]
Comment (gpei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2204]: ok, if so, we should leave them as p1, right?

[the remark this answers is not in the document]
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2204]: Yeah, once https://github.com/openshift/release/pull/34095 get merged, we need to modify most of job name to p1.

[the remark this answers is not in the document]
Comment (gpei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2204]: sure

**Pull request merged**